service/spring.gradle

```
@@ -1,28 +1,27 @@
 configurations { compileOnly { extendsFrom annotationProcessor } }
 
-var springBootVersion = '2.7.4'
-
+// All version controlled by dependency management plugin
 dependencies {
-	implementation "org.springframework.boot:spring-boot-starter-web:$springBootVersion"
-	developmentOnly "org.springframework.boot:spring-boot-devtools:$springBootVersion"
-	annotationProcessor "org.springframework.boot:spring-boot-configuration-processor:$springBootVersion"
-	testImplementation "org.springframework.boot:spring-boot-starter-test:$springBootVersion"
-	implementation "org.springframework.boot:spring-boot-starter-thymeleaf:$springBootVersion"
-
-	implementation 'org.apache.httpcomponents:httpclient:4.5.13'
+	implementation "org.springframework.boot:spring-boot-starter-actuator"
```

It's a minor thing but I think the gradle convention is to only use `""` for strings that have interpolation
```suggestion
	implementation 'org.springframework.boot:spring-boot-starter-actuator'
```

---

Thanks!  I'll clean up now that these no longer have interpolation.